### ISSUE TITLE
React dev tools unavailable

### ISSUE THREAD

**Issue by platinumitpl** [electron-react-boilerplate/electron-react-boilerplate#2547]
## Prerequisites

- [x] Using yarn
- [x] Using an up-to-date [`master` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/master)
- [x] Using latest version of devtools. [Check the docs for how to update](https://electron-react-boilerplate.js.org/docs/dev-tools/)
- [x] Tried solutions mentioned in [#400](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)
- [x] For issue in production release, add devtools output of `DEBUG_PROD=true yarn build && yarn start`

## Expected Behavior

The React tab should be available under the Chrome Developer Tool window.

## Current Behavior

React tab is not available. I can see Redux dev tools only.

## Steps to Reproduce

I have a standard configuration:

```javascript
const installExtensions = async () => {
  const installer = require('electron-devtools-installer');
  const forceDownload = !!process.env.UPGRADE_EXTENSIONS;
  const extensions = ['REACT_DEVELOPER_TOOLS', 'REDUX_DEVTOOLS'];

  return Promise.all(
    extensions.map((name) => installer.default(installer[name], forceDownload))
  ).catch(console.log);
};

```

## Context

I would like to investigate a component state.

## Your Environment

- Node version : v12.14.0
- electron-react-boilerplate version or branch : 1.3.0
- Operating System and version : macOS Catalina 10.15.5 (19F101)
- Link to your project :

**Comment by RafaelFontes** [comment]
In production release it is using the preload script parameter, therefore disabling some features. 

If you need to make it work on production for now, don't use preload option on webPreferences.

**Comment by platinumitpl** [comment]
The issue occurs in the dev env. `yarn dev`.
Perhaps I shouldn't have checked the last checkbox in the issue description.

**Comment by amilajack** [comment]
This issue isn't reproducible with a fresh clone. Please provide more details

**Comment by platinumitpl** [comment]
I get the same results on a fresh clone - missing React dev tools. 
What kind of details may help to track the bug?
I already posted software versions, so I don't see what else may help.
 
![image](https://user-images.githubusercontent.com/35427445/89877425-600eae00-dbc0-11ea-9083-ee241b9f25d4.png)

**Comment by amilajack** [comment]
Try running `UPGRADE_EXTENSIONS=1 yarn run dev`

**Comment by platinumitpl** [comment]
I read docs and tried that already, didn't help.

**Comment by shutterkurt** [comment]
I have the same issue on Windows 10 with a clean clone - unzip was throwing an exception.  looked into it and appears that the extension .crx file was empty due to a change on Google's end.  same issue as described by https://github.com/MarshallOfSound/electron-devtools-installer/pull/140.  This was fixed in 3.1.1 - I bumped up the version and 'seems' like it fixed the issue (didn't run any tests other than it doesn't throw the exception anymore and extension tabs show up now)
